tests/unit/test_tenant.py

```diff
@@ -12,30 +11,22 @@ def setup(self):
         self.custom_404_msg = "Tenant not found"
         self.schema = TenantSchema
 
-    def test_only_active_lease_returned(self, faker, create_lease, create_tenant):
-        tenant = create_tenant()
-
-        # Create an expired lease for the tenant
-        lease_expired_end = faker.date_time_this_decade(
-            before_now=True, after_now=False
-        )
-        lease_expired_start = lease_expired_end - datetime.timedelta(days=365)
-        lease_expired = create_lease(
-            tenant=tenant,
-            dateTimeStart=lease_expired_start,
-            dateTimeEnd=lease_expired_end,
-        )
-
-        # Create an active lease for the tenant
-        lease_active = create_lease(
-            tenant=tenant, property=PropertyModel.find(lease_expired.propertyID)
-        )
-
-        # Active lease should be the only one that shows up
-        assert tenant.json()["lease"] == lease_active.json()
-
```

Comment:
This is testing that only the active lease is serialized in the `.json` method. This should not be removed.

Reply:
This test doesn't belong here. I think it does need to be removed, but it needs to be replaced. This is testing the `active` method in `models/lease.py`. It's testing this through the json method. I wouldn't mind if this is done in this PR or we make another one for it. maybe the latter. But the `unit/test_lease` file should have a test confirming that its active method works appropriately. And then this can be removed.

To test that the `tenant.json` method returns the active lease then we would need to stub this method and test that it is called, as the json method shouldn't retest the functionality of the active method here.